### DISCUSSION TITLE
How certificates are generated in courses?

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Knowlg/Community#215] [General Discussions]
Hello @pallakartheekreddy, @HarishGangula, @swayangjit @vinukumar-vs @rajeevsathish, 

As I know, by enabling the **Issue Certificate** while creating a batch can generate certificate for the user.

But I want to understand the full flow the the certificate generation.

Like:
**1. Is it generated through telemetry? if yes, what is the cycle of certificate generation?**
**2. Does it get generated through code?**
**3. How we can enable the same process of certification generation like courses in _EVENTS_ as well**

cc:  @komalm @manishakapadnis @paritshivani

**Comment by pallakartheekreddy** [comment]
@nidhikachhawa24 certificates generation is part of Lern building block. Can you please create a thread in the below discussion forum.
https://github.com/orgs/Sunbird-Lern/discussions

**Reply by nidhikachhawa24** [reply]
Sure,

Can you tell me whom should I tag there?

@pallakartheekreddy
